flask/testing.py

```diff
@@ -48,6 +48,11 @@ class FlaskClient(Client):
 
     preserve_context = False
 
+    def __init__(self, *args, **kwargs):
+        environ_base = kwargs.pop("environ_base", None)
+        super(FlaskClient, self).__init__(*args, **kwargs)
+        self.environ_base = environ_base
```

Comment:
Are we switching from passing it in `app.test_client()` to env vars? I thought you meant in addition.